docs/modules/reference/pages/configuration/tuning-kafka.adoc

```diff
@@ -32,7 +32,7 @@ org.opennms.instance.id=MyNMS
 == Tips for Kafka
 
 * For Kafka RPC, the number of partitions should always be greater than the number of Minions at a location.
```

Comment:
```suggestion
* For Kafka RPC, the number of partitions should be a multiple of the number of Minions at a location.
```

Reply:
Does that mean at least a multiple of 2 of the number of minions at a location? For example, if there's only 2 minions at only one location, that'd be at least 4 partitions.

Reply:
2 is a multiple of 2.  If there are 2 minions, there should be 2, 4, 6, 8 ... partitions
If there are 3 minions, there should be 3, 6, 9, ... partitions

Reply:
Thanks for the clarification! @mmahacek